requirements.txt.in

```diff
@@ -1,6 +1,6 @@
 coloredlogs
 flatbuffers
 numpy >= @Python_NumPy_VERSION@
-packaging
+packaging >=22.0
```

Comment:
This change seems unnecessary. This file is to list what are needed when install a prebuilt ONNX Runtime package. Does it have this requirement?

Reply:
This arose when we created our own Docker containers for 5.4 and other items had requirements for packaging to be >= 22.0